### ISSUE TITLE
Starting more than 1 instance: data loading and database changes

### ISSUE THREAD

**Issue by khandelwal** [18F/foia-hub#687]
if we're starting more than one instance, we need to only have one of those instances do data loading and database changes.

**Comment by ramirezg** [comment]
I think we might be able to work around this issue by duplicating the app in the yaml file. The first only deploys one instance while updating the data and the second can deploy multiple instances, but doesn't update the database. 

manifest file
```
---
# all applications use these settings and services
memory: 1G
instances: 1
applications:
- name: openfoia-staging
  path: .
  timeout: 180
  command: echo '---loading data---' & bash cf.sh
- name: openfoia-staging
  instances: 2
  path: .
  timeout: 180
  command: echo '---deploying---' & waitress-serve --port=$VCAP_APP_PORT foia_hub.wsgi:application

```

**Comment by arowla** [comment]
We solved a similar problem on FEC by using the CF_INSTANCE_INDEX
https://github.com/18F/openFEC/blob/master/run_api.py

**Comment by ramirezg** [comment]
@arowla This is great, thanks!

**Comment by khandelwal** [comment]
:heart: @arowla for jumping in and showing us how it's done. We're using your solution.

**Comment by arowla** [comment]
Glad y'all appreciated the drive-by. :car:

**Comment by ramirezg** [comment]
Solved with 18F/foia-hub/pull/742
